### PR TITLE
i18n(es): translate `file-tree`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Translates `file-tree` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/components/file-tree.mdx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+Para mostrar la estructura de un directorio con iconos de archivo y subdirectorios colapsables, usa el componente `<FileTree>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<FileTree slot="preview">
+
+- astro.config.mjs an **important** file
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/file-tree.mdx:16]: We can safely translate the "an important file" comment in the preview.

File: docs/src/content/docs/es/components/file-tree.mdx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+{% /filetree %}
+```
+
+</Fragment>
+
+<FileTree slot="preview">
+
+- src
+  - components
+    - Header.astro an **important** file
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/file-tree.mdx:170]: Same comment for the 3 instances in this example.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left 1 comment regarding an example that could be translated but other than that, the translation looks good to me.

**Comment by dreyfus92** [comment]
Nice catch @HiDeoo, all suggestions have been applied 😁

**Review by HiDeoo — APPROVED** [review]
Thanks for the fixes, looks perfect to me now :rocket:

**Pull request merged**